templates/homepage.html.twig

```diff
@@ -0,0 +1,388 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <meta name="description" content="" />
+        <meta name="author" content="" />
+        <title>Bibliothèque</title>
+        <!-- Favicon-->
+        <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
+        <!-- Font Awesome icons (free version)-->
+        <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
+        <!-- Google fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css" />
+        <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css" />
```

Comment:
Could you try to install these dependencies locally using NPM instead of relying on free CDNs please? These CDNs (especially Google one's) cause privacy issues as they are used to track end users.

Google fonts are available as NPM packages thanks to [fontsource](https://fontsource.org/) and Font Awesome has [official NPM packages](https://fontawesome.com/fontawesome.com/docs/web/setup/packages).